src/TranscriptRow/rows.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import moment from 'moment';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import {
+  faPlay,
+  faVolumeUp,
+  faSyncAlt,
+  faLayerGroup
+} from '@fortawesome/free-solid-svg-icons';
+
+moment().format();
+
+const getExpiryDate = (createdDate) => {
+  const dateCreated = moment(new Date(createdDate));
+  const dateNow = moment();
+  const expiryDate = moment(new Date(dateCreated)).add(60, 'days');
+  const daysUntilExpiry = expiryDate.diff(dateNow, 'days');
+
+  return daysUntilExpiry;
+};
+
+const formatMessage = (message, mediaDuration) => {
+  if (message === 'Transcribing...') {
+    return `Transcribing${ mediaDuration ? `, approx. ${ mediaDuration }` : '...' }`;
```

Comment:
got a little confused by the ternary inside a template literal, this could be a little easier to read:
```return mediaDuration ? `Transcribing, approx. ${ mediaDuration }` : message;```

think this is just a personal preference so feel free to disregard 🙂

Reply:
No you're right, this would make it cleaner! Thanks :)